.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -117,13 +117,15 @@ jobs:
             external-kvstore: false
             max-connected-clusters: 255
             cm-auth-mode: 'legacy'
+            policy-default-local-cluster: true
```

Comment:
I would avoid enabling this option in the upgrade test for the moment, given that it does not provide additional coverage, and it is potentially confusing, because it implicitly depends on the fact that the second cluster supports it.